### PR TITLE
testing pr build

### PR DESCRIPTION
**Pull request by murillodigital**
DO NOT MERGE - THIS IS JUST FOR TESTING PURPOSES

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
 //
```
Comment (murillodigital) [inline interface/src/Application.cpp]: This is just a test

**Comment by murillodigital** [comment]
test

**Comment by murillodigital** [comment]
This is the comment in the PRhttps://gist.github.com/454ee4db7372d27c4f0d

**Comment by murillodigital** [comment]
test this please

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
This is the comment in the PR: https://gist.github.com/f46adef04406ee0fd75a

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 11f07216de382083e9ec2e3e8ab610e3557bf56a available at: https://gist.github.com/9d837efcdb79b63ab84f

**Pull request closed without merging**